airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,6 +24,18 @@
 Changelog
 ---------
 
+6.3.0
+-----
+
+Features
+~~~~~~~~
+
+Pandas is now an optional dependency of the provider. The ``SqlToS3Operator`` and ``HiveToDynamoDBOperator``
+require Pandas to be installed (you can install it automatically by adding ``[pandas]`` extra when installing
+the provider. Those who installed previous version of the provider, should already have pandas installed, so
+this is not a breaking change.
```

Comment:
Just wondering, should we also mentioned `RedshiftSQLHook` methods `get_pandas_df` and `get_pandas_df_by_chunks`? However this methods provided by `DbApiHook` so maybe not.

Reply:
That's a good point - those are local imports there and they   `raise Exception(` but now we can change them to the OptionalProviderFeature exception. I will do that as a follow-up.